test/smoke-limbo/omptest-device-emi/Makefile

```diff
@@ -0,0 +1,24 @@
+include ../../Makefile.defs
+
+# This particular test will primarily use a bash script to handle the build via
+# cmake and actual execution.
+# Hence, we build a dummy target to remain compliant.
+
+TESTNAME     = omptest-device-emi
+TESTSRC_MAIN =
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+DUMMYSRC     = echo 'int main(void) { return 0; }'
+
+RUNCMD       = ./performBuildRun.sh -a $(AOMP) -t $(AOMP_GPU)
+
+CLANG        ?= clang
+OMP_BIN      = $(AOMP)/bin/$(CLANG)
+RUNENV       += OMPTEST_USE_OMPT_EMI=1 OMPTEST_USE_OMPT_TRACING=1 OMPTEST_RUN_AS_TESTSUITE=1 CPLUS_INCLUDE_PATH=$(AOMP)/lib64/omptest/include
```

Comment:
nit: Just realized: the `CPLUS_INCLUDE_PATH` has no effect (since there is no `lib64` directory).

Also true for: smoke-limbo/veccopy-ompt-target-cmake/Makefile
IMO I can clean this up once both PRs have landed.

Reply:
yeah, let's land this and then solve these nits.